File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>
int main()
{
    int height;
    do
    {
        printf("Height: \n");
        height=get_int();
        
    } while(height <=0 || height >=24);
    
    for(int rows=0;rows<height;rows++)
    {
        for(int spaces=0;spaces < height-rows-1;spaces++ )
        {
            printf("%s", " ");
        }
        for (int hashes= 0; hashes < rows+2; hashes++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height: ", end = "")
        height = cs50.get_int()
        if height >= 0 and height <= 23:
            break
        
    for row in range(height):
        for space in range(height - row - 1):
            print(" ", end = "")
        for hash in range(row+2):
            print("#", end = "")
        print("")
            
if __name__ == "__main__":
    main()
